warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/handler/shard/ShardedDataTypeHandler.java

```diff
@@ -1019,50 +1019,52 @@ protected Multimap<BulkIngestKey,Value> createShardFieldIndexColumn(RawRecordCon
 
         Multimap<BulkIngestKey,Value> values = HashMultimap.create();
 
-        Text colf = new Text("fi");
-        TextUtil.textAppend(colf, fieldName, replaceMalformedUTF8);
-        Text unmaskedColq = new Text(fieldValue);
-        TextUtil.textAppend(unmaskedColq, event.getDataType().outputName(), replaceMalformedUTF8);
-        TextUtil.textAppend(unmaskedColq, event.getId().toString(), replaceMalformedUTF8);
-
-        if (value == null) {
-            value = NULL_VALUE;
-        }
-
-        if (null != maskedFieldHelper && maskedFieldHelper.contains(fieldName)) {
-            if (!StringUtils.isEmpty(fieldValue)) {
-                // Put unmasked colq with original visibility
-                Key k = createKey(shardId, colf, unmaskedColq, visibility, event.getDate(), deleteMode);
-                BulkIngestKey bKey = new BulkIngestKey(this.getShardTableName(), k);
-                values.put(bKey, value);
+        if (!StringUtils.isEmpty(fieldValue)) {
+            Text colf = new Text("fi");
+            TextUtil.textAppend(colf, fieldName, replaceMalformedUTF8);
+            Text unmaskedColq = new Text(fieldValue);
+            TextUtil.textAppend(unmaskedColq, event.getDataType().outputName(), replaceMalformedUTF8);
+            TextUtil.textAppend(unmaskedColq, event.getId().toString(), replaceMalformedUTF8);
+
+            if (value == null) {
+                value = NULL_VALUE;
             }
 
-            // We need to use the normalized masked values
-            final String normalizedMaskedValue = helper.getNormalizedMaskedValue(fieldName);
-            if (!StringUtils.isEmpty(normalizedMaskedValue)) {
-                Text maskedColq = new Text(normalizedMaskedValue);
-                TextUtil.textAppend(maskedColq, event.getDataType().outputName(), replaceMalformedUTF8);
-                TextUtil.textAppend(maskedColq, event.getId().toString(), replaceMalformedUTF8);
+            if (null != maskedFieldHelper && maskedFieldHelper.contains(fieldName)) {
+                if (!StringUtils.isEmpty(fieldValue)) {
```

Comment:
Do we still need this with the check above?

Reply:
No we do not.